### ISSUE TITLE
sentry: crash_reporting.go:471: txn: unexpected 133079040 leftover bytes | string

### ISSUE THREAD

**Issue by tbg** [cockroachdb/cockroach#31958]
https://sentry.io/cockroach-labs/cockroachdb/issues/745624517/

crash_reporting.go:471: txn: unexpected 133079040 leftover bytes | string

github.com/cockroachdb/cockroach/pkg/util/mon.(*BytesMonitor).doStop

> stacktrace: {u'frames': [{u'function': u'func3', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go', u'module': u'github.com/cockroachdb/cockroach/pkg/sql/pgwire.(*conn).serveImpl', u'in_app': False, u'lineno': 260, u'filename': u'github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go'}, {u'function': u'ServeConn', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go', u'module': u'github.com/cockroachdb/cockroach/pkg/sql.(*Server)', u'in_app': False, u'lineno': 491, u'filename': u'github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go'}, {u'function': u'run', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go', u'module': u'github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor)', u'in_app': False, u'lineno': 1029, u'filename': u'github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go'}, {u'function': u'txnStateTransitionsApplyWrapper', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go', u'module': u'github.com/cockroachdb/cockroach/pkg/sql.(*connExecutor)', u'in_app': False, u'lineno': 1650, u'filename': u'github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go'}, {u'function': u'ApplyWithPayload', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/util/fsm/fsm.go', u'module': u'github.com/cockroachdb/cockroach/pkg/util/fsm.(*Machine)', u'in_app': False, u'lineno': 136, u'filename': u'github.com/cockroachdb/cockroach/pkg/util/fsm/fsm.go'}, {u'function': u'apply', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/util/fsm/fsm.go', u'module': u'github.com/cockroachdb/cockroach/pkg/util/fsm.Transitions', u'in_app': False, u'lineno': 107, u'filename': u'github.com/cockroachdb/cockroach/pkg/util/fsm/fsm.go'}, {u'function': u'func11', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/sql/conn_fsm.go', u'module': u'github.com/cockroachdb/cockroach/pkg/sql.glob.', u'in_app': False, u'lineno': 380, u'filename': u'github.com/cockroachdb/cockroach/pkg/sql/conn_fsm.go'}, {u'function': u'finishSQLTxn', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/sql/txn_state.go', u'module': u'github.com/cockroachdb/cockroach/pkg/sql.(*txnState2)', u'in_app': False, u'lineno': 221, u'filename': u'github.com/cockroachdb/cockroach/pkg/sql/txn_state.go'}, {u'function': u'Stop', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go', u'module': u'github.com/cockroachdb/cockroach/pkg/util/mon.(*BytesMonitor)', u'in_app': False, u'lineno': 384, u'filename': u'github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go'}, {u'function': u'doStop', u'abs_path': u'/go/src/github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go', u'module': u'github.com/cockroachdb/cockroach/pkg/util/mon.(*BytesMonitor)', u'in_app': False, u'lineno': 398, u'filename': u'github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go'}]}
type: *log.safeError
value: crash_reporting.go:471: txn: unexpected 133079040 leftover bytes | string

**Comment by jordanlewis** [comment]
This is in v2.0.6.

```
github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go in doStop at line 398
github.com/cockroachdb/cockroach/pkg/util/mon/bytes_usage.go in Stop at line 384
github.com/cockroachdb/cockroach/pkg/sql/txn_state.go in finishSQLTxn at line 221
github.com/cockroachdb/cockroach/pkg/sql/conn_fsm.go in func11 at line 380
github.com/cockroachdb/cockroach/pkg/util/fsm/fsm.go in apply at line 107
github.com/cockroachdb/cockroach/pkg/util/fsm/fsm.go in ApplyWithPayload at line 136
github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go in txnStateTransitionsApplyWrapper at line 1650
github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go in run at line 1029
github.com/cockroachdb/cockroach/pkg/sql/conn_executor.go in ServeConn at line 491
github.com/cockroachdb/cockroach/pkg/sql/pgwire/conn.go in func3 at line 260
```

**Comment by jordanlewis** [comment]
Hopeful this is solved by #34935.
